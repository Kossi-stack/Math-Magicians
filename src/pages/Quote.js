const Quote = () => (
  <p className="quoteParagraph">
    A computer would deserve to be called intelligent
    if it could deceive a human into believing that it was human.
    {' '}
    { ' ' }
    {' '}
    <b>

      -Alan Turing

    </b>
    {' '}

  </p>
);

export default Quote;
