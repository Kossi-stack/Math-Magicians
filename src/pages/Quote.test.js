import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
import Quote from './Quote';

it('Quote component is working', () => {
  const tree = renderer
    .create(
      <Router>
        <Switch>
          <Route>
            <Quote />
          </Route>
        </Switch>
      </Router>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
