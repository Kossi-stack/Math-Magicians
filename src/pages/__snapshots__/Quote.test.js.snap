// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component is working 1`] = `
<div
  className="container"
>
  <h3>
    Mathematics is a language. â€” Josiah Willard Gibbs, American scientist
  </h3>
  <h3>
    If you stop at general math, then you will only make general money. - Snoop Dogg
  </h3>
  <h3>
    Pure mathematics is, in its way, the poetry of logical ideas. - Albert Einstein
  </h3>
</div>
`;
